cmd/kubeadm/app/cmd/phases/init/kubeconfig.go

```diff
@@ -74,7 +74,7 @@ func NewKubeConfigPhase() workflow.Phase {
 				RunAllSiblings: true,
 			},
 			NewKubeConfigFilePhase(kubeadmconstants.AdminKubeConfigFileName),
-			NewKubeConfigFilePhase(kubeadmconstants.KubeletKubeConfigFileName),
+			NewKubeConfigFilePhase(kubeadmconstants.KubeletBootstrapKubeConfigFileName),
```

Comment:
are we able to change the cli, without a deprecation cycle?

Reply:
the phase command still remains `kubelet`, but this does change the target file.